zanata-war/src/main/webapp/WEB-INF/layout/version/merge_trans_modal.xhtml

```diff
@@ -158,7 +158,7 @@
               action="#{mergeTransAction.startMergeTranslations}"
               value="#{msgs['jsf.iteration.mergeTrans']}"/>
 
-            <ui:fragment rendered="#{mergeTransAction.isCopyActionsRunning(mergeTransAction.targetProjectSlug, mergeTransAction.targetVersionSlug)}">
+            <ui:fragment rendered="#{mergeTransAction.isCopyActionsRunning()}">
```

Comment:
:+1: Good find, but this looks like it's fixing a different issue. Not sure if we should wait for CDI to also fix this.